packages/ethereum/contracts/src/Crypto.sol

```diff
@@ -677,19 +677,11 @@ abstract contract HoprCrypto {
    * @param DST domain separation tag, used to make protocol instantiations unique
    */
   function expand_message_xmd_keccak256(bytes memory message, bytes memory DST) internal pure returns (bytes32 b_1, bytes32 b_2, bytes32 b_3) {
-    uint256 ell; 
```

Comment:
Why can you skip these checks?

Reply:
incorrectly implemented, only applies to general implementation without fixed output length